piqe_ocp_lib/tests/resources/test_ocp_health_checker.py

```diff
@@ -23,15 +23,15 @@ def test_check_node_health(self, ocp_health):
         all_nodes_healthy, node_health_info = ocp_health.check_node_health()
         assert isinstance(all_nodes_healthy, bool)
         assert isinstance(node_health_info, dict)
-        # if all_nodes_healthy is False, There has to be values of failure nodes in node_health_info dict
+        # if all_nodes_healthy is False, There has to be >1 values of failure nodes in node_health_info dict
         if all_nodes_healthy:
             for key, value in node_health_info.items():
-                assert len(value) == 0
+                assert len(value) == 1
```

Comment:
If all nodes are healthy, It means no failure and hence all values in node_health_info should be empty.

Reply:
Since we are adding 'NodeReadyStatus' to 'temp_list' in all cases, we will have a false negative result if we leave '                assert len(value) == 0'

Reply:
Added conditional in ocp_health_checker.py to only add NodeReadyState to 'unhealthy_node_info' if all_nodes_healthy = False, so we can assert len(value) == 0